examples/utilities.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+
+def ensure_no_files_overwritten(expected_contents):
+    # ensure we don't overwrite the user's files, if they run it on their machine
+    if os.path.exists(os.path.join(os.getcwd(), 'example.txt')):
```

Comment:
We call `open(...)` with just the basename below so we should be able to do the same here and drop the os.path.join parts.

Reply:
Yeah fair, I just didn't want to worry about if Windows would throw a fit at doing that. I'll fix that now.